lib/appraisal/appraisal.rb

```diff
@@ -118,8 +118,12 @@ def update_command(gems)
       ['bundle', 'update', *gems].compact
     end
 
+    def project_root
```

Comment:
Super pedantic, but do you mind moving this method below the `#gemfile_root` definition? That way the file can read more like a newspaper article -- giving more details as you go -- so we don't have to read a method and then wonder what it might be used for in the future.

Reply:
Yes, I like that!